src/testbed.cxx

```diff
@@ -186,12 +198,12 @@ int main(int argc, char *argv[])
 				"double\t||x||_2^2(dim=" + std::to_string(DIM)+")",
 				t.approx, t.exact);
 #endif
-		auto end = std::chrono::steady_clock::now();
-    		std::chrono::duration<double> elapsed_seconds = end-start;
-    		std::cout << "elapsed time: " << elapsed_seconds.count() << "s\n";
-		
+		end = std::chrono::steady_clock::now();
+		elapsed_seconds = end-start;
+		print_timer("chrono\tNUM_TRIALS", elapsed_seconds);
```

Comment:
"Type" is the operation performed. Saying just "chrono" here isnot informative, use for example, the same string I had here: `"double\t||x||_2^2(dim=" + std::to_string(DIM)+")"`

Reply:
Similary, NUM_TRIALS is not helpful. Please print out the actual number of trials. This is probably better as a field as another entry. That is, the header should be

`Type\tOperation\tTrials\tTime(s)`

And so your print_timer should take in 3 arguments.